ViAn/Project/videoproject.cpp

```diff
@@ -126,14 +126,17 @@ void VideoProject::read(const QJsonObject& json){
     for (int j = 0; j < json_analyses.size(); ++j) {
         QJsonObject json_analysis = json_analyses[j].toObject();        
         BasicAnalysis* analysis;
-        SAVE_TYPE save_type = (SAVE_TYPE)json_analysis["save_type"].toInt();
+        ANALYSIS_TYPE save_type = (ANALYSIS_TYPE)json_analysis["analysis_type"].toInt();
         switch(save_type){
-        case INTERVAL:
+        case TAG:
             analysis = new Tag();
             break;
-        case DETECTION:
+        case MOTION_DETECTION:
```

Comment:
I think `DETECTION` was used here since it would make it simpler if more detection types are added